client/components/hello-world.jsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
+import BookList from '../components/books-list';
 
-export default function HelloWorld(props) {
-  return (
-    <h1>Hello, World!</h1>
-  );
+export default class HelloWorld extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      books: []
+    };
+  }
+
+  componentDidMount() {
+    const url = 'https://api.nytimes.com/svc/books/v3/lists/current/hardcover-fiction.json?api-key=oN0z6slOegD5zjvuGpGHt1uNGJS1uEGE';
```

Comment:
You should put the api key in the .env file and use `${process.env.BOOKS_API_KEY}` instead of hard-coding it.

Reply:
You can make the BOOKS_API_KEY work by updating webpack.config.js


https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs